compiler/rustc_trait_selection/src/traits/error_reporting/suggestions.rs

```diff
@@ -741,7 +741,7 @@ impl<'a, 'tcx> InferCtxtExt<'tcx> for InferCtxt<'a, 'tcx> {
 
                     let msg = format!(
                         "the trait bound `{}: {}` is not satisfied",
-                        orig_ty.to_string(),
+                        orig_ty,
```

Comment:
This one is fixing a different issue, but that's fine, it's also a good change :)

Reply:
nothing to see here, please move along :p